addons/arsenal/functions/fnc_addRightPanelButton.sqf

```diff
@@ -63,11 +63,9 @@ private _configItemInfo = "";
 _items = _items select {
     _configItemInfo = _cfgWeapons >> _x >> "ItemInfo";
 
-    _x isKindOf ["CBA_MiscItem", _cfgWeapons] && {getNumber (_configItemInfo >> "type") in [TYPE_MUZZLE, TYPE_OPTICS, TYPE_FLASHLIGHT, TYPE_BIPOD]} ||
+    _x call FUNC(isMiscItem) && {getNumber (_configItemInfo >> "type") in [TYPE_MUZZLE, TYPE_OPTICS, TYPE_FLASHLIGHT, TYPE_BIPOD]} ||
```

Comment:
Conditions could be swapped around, the single config lookup is probably faster than the whole function call, but I haven't tested it.